clippy_lints/src/doc/markdown.rs

```diff
@@ -97,7 +104,7 @@ fn check_word(cx: &LateContext<'_>, word: &str, span: Span, code_level: isize) {
     }
 
     // We assume that mixed-case words are not meant to be put inside backticks. (Issue #2343)
```

Comment:
Could you add a comment like this one, but mentioning this issue? Meow meow :cat:
```suggestion
    // We assume that mixed-case words are not meant to be put inside backticks. (Issue #2343)
    // We also assume that quotations should not be changed from the original source, so we will not lint in those. (Issue #10262)
```

Reply:
Good idea! Done.